### DISCUSSION TITLE
Dreame bot D10s

### DISCUSSION THREAD

**Discussion by Giobbe85** [Tasshack/dreame-vacuum#106] [Q&A] [answered]
Hello everyone,
is there a possibility to insert the D10s ? ( it is not present even in MI HOME yet ) I saw that the D10 pro is present.
thanks a lot for your patience

**Comment by Tasshack** [comment, marked as answer]
If your device is not available on Mi Home App you cannot get the api token for communicating the device therefore you cannot use this integration until you add your device to your Mi Home account.

**Comment by DavidKaub** [comment]
@Giobbe85 Are there any updates for availability of the D10s in Mi HOME? I'm currently in the process of buying a vacuum, but would like one that has no camera but is compatible to HA

**Reply by Tasshack** [reply]
You could choose which robot to buy by referring the https://github.com/Tasshack/dreame-vacuum#supported-devices

**Reply by uusitalo** [reply]
Is D9 Max same as D9 Max white?

**Comment by Tasshack** [comment]
https://github.com/Tasshack/dreame-vacuum/discussions/109
